analyzers/dataframe/CMakeLists.txt

```diff
@@ -17,11 +17,25 @@ include_directories(${EDM4HEP_INCLUDE_DIRS}
                     ${VDT_INCLUDE_DIR}
                     )
 
-message(STATUS "includes-------------------------- dataframe awkward: ${AWKWARD_INCLUDE}")
-message(STATUS "includes-------------------------- dataframe edm4hep: ${EDM4HEP_INCLUDE_DIRS}")
-message(STATUS "includes-------------------------- dataframe podio  : ${podio_INCLUDE_DIR}")
-message(STATUS "includes-------------------------- dataframe fastjet: ${FASTJET_INCLUDE_DIRS}")
-
+message(STATUS "includes-------------------------- dataframe awkward:     ${AWKWARD_INCLUDE}")
+message(STATUS "includes-------------------------- dataframe edm4hep:     ${EDM4HEP_INCLUDE_DIRS}")
+message(STATUS "includes-------------------------- dataframe podio  :     ${podio_INCLUDE_DIR}")
+message(STATUS "includes-------------------------- dataframe fastjet:     ${FASTJET_INCLUDE_DIRS}")
+
+set(EXTRA_INCLUDE_DIRS)
+set(EXTRA_LIBRARIES)
+set(EXTRA_ROOT_INCLUDES)
+
+find_package(ONNXRuntime)
```

Comment:
For packaging, it is preferred to have an explicit option to build the onnx dependent parts, and to fail if they cannot be found. Could you add a `WITH_ONNX` option similar to `WITH_DD4HEP` https://github.com/HEP-FCC/FCCAnalyses/blob/ff8ff034e8d3524ec0daeb224ccb46c49a293cd2/CMakeLists.txt#L18 ?

Reply:
Thanks for your review, @vvolkl. Sure, will do!